apps/meteor/server/models/raw/LivechatContacts.ts

```diff
@@ -9,6 +10,29 @@ export class LivechatContactsRaw extends BaseRaw<ILivechatContact> implements IL
 		super(db, 'livechat_contact', trash);
 	}
 
+	protected modelIndexes(): IndexDescription[] {
+		return [
+			{
+				key: { name: 1 },
+				unique: false,
+				sparse: true,
+				name: 'name_insensitive',
+				collation: { locale: 'en', strength: 2, caseLevel: false },
+			},
+			{
+				key: { emails: 1 },
+				unique: false,
+				sparse: true,
+				name: 'emails_insensitive',
+				collation: { locale: 'en', strength: 2, caseLevel: false },
+			},
+			{
+				key: { phones: 1 },
+				sparse: true,
```

Comment:
```suggestion
				sparse: true,
				unique: false,
```
Keeping the `unique: false` here just for visibility. The others indexes have it, so this one seemed like it was different, but the default value is also false.